libs/sdk-ui-dashboard/src/model/events/userInteraction.ts

```diff
@@ -21,6 +21,34 @@ export type KpiAlertDialogOpenedPayload = UserInteractionPayloadWithDataBase<
     }
 >;
 
+/**
+ * @alpha
+ */
+export type DescriptionTooltipOpenedFrom = "kpi" | "widget" | "insight";
+/**
+ * @alpha
+ */
+export type DescriptionTooltipOpenedType = "inherit" | "custom";
+/**
+ * @alpha
+ */
+export type DescriptionTooltipOpenedData = {
+    from: DescriptionTooltipOpenedFrom;
+    type: DescriptionTooltipOpenedType;
+    descriptionLength: number;
+};
+/**
+ * @alpha
+ */
+export type DescriptionTooltipOpenedPayload = UserInteractionPayloadWithDataBase<
+    "descriptionTooltipOpened",
+    {
+        from: DescriptionTooltipOpenedFrom;
```

Comment:
It should be probably `DescriptionTooltipOpenedPayload = UserInteractionPayloadWithDataBase<
    "descriptionTooltipOpened",  DescriptionTooltipOpenedData >` ?

Reply:
you are right, I have created  DescriptionTooltipOpenedData additionally and forget to use it here